build/DependencyVersions.props

```diff
@@ -1,34 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_SharedFrameworkVersion>2.0.3</CLI_SharedFrameworkVersion>
```

Comment:
This change will require an update on dotnet/versions, so that the right things are automatically update here.